scripts/start-db-in-docker.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+set -e -u
+
+SCRIPT_PATH="$(cd "$(dirname "${0}")" && pwd)"
+
+function bootDB {
+  db="$1"
+
+  if [ "${db}" = "postgres" ]; then
+    launchDB="(/postgres-entrypoint.sh postgres &> /var/log/postgres-boot.log) &"
+    testConnection="psql -h localhost -U postgres -c '\conninfo'"
+  elif [ "${db}" = "mysql" ]  || [ "${db}" = "mysql-5.6" ] || [ "${db}" = "mysql8" ]; then
```

Comment:
In `scripts/docker-shell-with-started-db` the `db` variable will be set to `mysql` if e.g. `DB=mysql8`; so the two `OR`s are not needed.